MailCore/Cache/ContactManager/ContactManager.swift

```diff
@@ -20,12 +20,16 @@ import CocoaLumberjackSwift
 import Contacts
 import Foundation
 import InfomaniakCore
+import InfomaniakCoreDB
 import InfomaniakCoreUI
 import RealmSwift
 import SwiftRegex
 
 /// The composite protocol of the `ContactManager` service
-public typealias ContactManageable = ContactFetchable & ContactManagerCoreable & RealmAccessible
+public typealias ContactManageable = ContactFetchable
+    & ContactManagerCoreable
+    & RealmConfigurable
```

Comment:
Now the contact manager is only `RealmConfigurable` and `Transactionable`. It no longer provides a way to call `getRealm()` directly, since you should not do it anymore.